.gitmodules

```diff
@@ -3,7 +3,13 @@
         url           = https://github.com/NCAR/ccpp-framework
         fxtag         = 2024-07-19-dev
         fxrequired    = AlwaysRequired
-        fxDONOTUSEurl = https://github.com/NCAR/ccpp-framework
+	fxDONOTUSEurl = https://github.com/NCAR/ccpp-framework
+[submodule "history"]
+	path = src/history/buffers
+	url = https://github.com/peverwhee/history_output
+	fxtag         = b1da47af645dea3b14c19efc0716dc6ce6c28d22
+	fxrequired    = AlwaysRequired
+	fxDONOTUSEurl = https://github.com/peverwhee/history_output
```

Comment:
I assume these will be updated to "official" links and not a private fork

Reply:
updated!